### ISSUE TITLE
[Bug] Scheduler task throws error

### ISSUE THREAD

**Issue by typoniels** [eliashaeussler/typo3-warming#1]
When running the planner task with the sites field selected I get the following error message when running, apparently getApplication does not provide any data here.The execution via the CLI works fine.

TYPO3: 10.4.15
PHP 7.4.18
Composer: Yes

![screencapture-typoniels-de-typo3-index-php-2021-05-10-17_00_06](https://user-images.githubusercontent.com/53429393/117689986-9ebaae80-b1ba-11eb-91d9-51509e1e9288.png)

**Comment by eliashaeussler** [comment]
Thanks for this bug report, @typoniels!

I pushed a fix to this bug. You can test it by running

```bash
composer req eliashaeussler/typo3-warming:"dev-develop as 0.2.2"
```

I hope it works for you. Appreciate your feedback :)

**Comment by eliashaeussler** [comment]
I just released a new version containing the bugfix: 0.2.2

Thanks again for reporting!
